Makefile

```diff
@@ -145,4 +159,17 @@ test-single: export COOKIE_ENC_KEY="65d31d12b80fc57df16d84c02a9bb62e2bc3b633388b
 test-single:
 	GORACE='$(racevars)' go test -race -v -tags='testing debug netgo' -timeout=300s $(integration-pkgs) -run=$(RUN) -count=$(count)
 
-.PHONY: all fmt install release clean check test test-int test-long test-single start-mongo stop-mongo
+# docker-genenv is a docker command for env var generation
+#
+# The sleep is to handle a slight delay in the jwks.json file being generated in
+# the docker container. Without it the file is blank.
+docker-genenv: clean
+	@mkdir docker
+	@docker build . -t accounts-genenv
+	@docker run --name genenv -d accounts-genenv
+	sleep 3
+	@docker cp genenv:/app/env ./docker/env
+	@docker cp genenv:/app/jwks.json ./docker/jwks.json
+	@docker stop genenv || true && docker rm genenv || true
```

Comment:
I get what you are doing here but I wonder whether we can achieve the same in a cleaner way if we touch the files before the run and then mount them as volumes. This will allow us to get rid of the `for {}` and we won't need a `sleep 3` either - the run will finish as soon as it's done without any risk of being stopped too early.

Reply:
So the sleep is really a factor of there needing to be time for the docker container to start up. 
The docker run command returns instantly and then we are immediately issuing docker stop. 
I removed the `for {}` in the main.go file as that actually didn't do anything.

I think the sleep is the simplest way to consistently run this command without error.

Reply:
Usually when the entry point of a docker container exits, the entire docker container exits as well. If the container is marked with `-rm` then its image is also removed (I might be mixing up `docker` and `docker-compose`). So, if the entry point binary does all it needs to do and exits with 0 then we shouldn't need to call `stop` and `rm`.

In any case, this is a secondary deliberation. The current state should be good enough for what we need. I was worried that three seconds might not be enough on certain systems and that we'd get weird and hard to debug error reports about missing JWKS and such but I might be overthinking there.